### ISSUE TITLE
Feature request - Volume adjustment for ambient sounds + even moar ambience 

### ISSUE THREAD

**Issue by NZF-JDWang** [ALiVEOS/ALiVE.OS#638]
### Ambient sound volume

The ambient sounds are perfect for small villages etc but if you want to make say Zargabad a really busy sounding place it would be great if we had say a 1-5 volume control on the module just so you could crank that volume for certain missions.

### Moar ambience 

Now the only thing I miss from TPW mods is the ambient flybys

I'd lump this in with the civilian side of things because the aircraft should all be setcaptive to avoid unwanted mission interaction, but it'd be great to have a field where I could specify an array of aircraft and see them flying across the map every so often at different heights and decent speeds. If nothing else it would greatly add to the ambient soundscape.

**Comment by diveyez** [comment]
Yeah. This has been requested for a long time. The issue with this is everyone loves the loud crying baby. LOL.

**Comment by NZF-JDWang** [comment]
Well now tup has provided a way to remove said baby sounds https://github.com/ALiVEOS/ALiVE.OS/issues/722

Ran a zeus mission for the lad the other night using a zeus soundpack and one of the big comments after the mission was how much more immersive it felt with all the busy market sounds etc.

They were essentially the same types of sounds as ALiVE, but at a higher volume. I just really feel the default sounds are much too quiet and it would be great to be able to boost them depending on the style of mission.
